### DISCUSSION TITLE
[tabulator-tables] HistoryAction "RowMoved" should be "RowMove"

### DISCUSSION THREAD

**Discussion by Deejerz88** [DefinitelyTyped/DefinitelyTyped#66572] [Issues with a @types package]
From tabulator-tables History module:

![image](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/103090951/adf3170c-4eac-4593-a619-39c5a3150c5c)

[a comment by an automated account is not shown]

**Comment by mike-lischke** [comment]
@Deejerz88 Can you point us to the relevant portion of the online documentation?

**Reply by Deejerz88** [reply]
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/tabulator-tables/index.d.ts

ln. 1170 HistoryAction

https://tabulator.info/docs/5.5/events#history

He actually calls it "rowMoved" in the documentation, but in his src code he labels the action "rowMove" as in my screenshot and confirmed by logging the action when it occurs.

I have reached out to Oli to let him know of the typo in the documentation as well.

**Reply by mike-lischke** [reply]
Oh interesting. Can you file a PR once the documentation is fixed? Would be confusing as well, if we change the types and they differ to the documentation.
